pynest/nest/lib/hl_api_connections.py

```diff
@@ -225,6 +225,11 @@ def Connect(pre, post, conn_spec=None, syn_spec=None,
             raise ValueError("When connecting two arrays of node IDs, the synapse specification dictionary must "
                              "be specified and contain at least the synapse model.")
 
+        if "weights" in processed_syn_spec:
```

Comment:
Is this here in case 'weight' is misspelled?

Reply:
Yes. I've added a comment.